### PR TITLE
Took out some un needed stuff

### PR DESCRIPTION
**Pull request by Addison118**

### CODE REVIEW

**Comment by Addison118** [comment]
Whoops... Forgot to change the setCurrentChatState

File: src/legionpe/theta/Session.php [hunk 1]
```diff
@@ -930,6 +930,9 @@ public function joinChannel($channel, $subLevel = self::CHANNEL_SUB_NORMAL){
 		new JoinChannelQuery($this->getMain(), $this->getUid(), $channel, $subLevel);
 		return true;
 	}
+    public function getCurrentChatState() {
+        return $this->currentChatState;
+    }
```
Comment (PEMapModder) [inline src/legionpe/theta/Session.php]: Please use tabs

File: src/legionpe/theta/command/session/ChannelCommand.php [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        }
+        if (strtolower($args[0]) == "team") {
+            if($session->getCurrentChatState() === Session::CHANNEL_TEAM) {
+                $session->getPlayer()->sendMessage(TextFormat::RED . "The channel state is already the team");
+            } else {
+                $session->currentChatState = Session::CHANNEL_TEAM;
+            }
+        }
+        $channel = array_shift($args);
+        $session->joinChannel($channel);
```
Comment (PEMapModder) [inline src/legionpe/theta/command/session/ChannelCommand.php]: You need an else here.

**Comment by Addison118** [comment]
@PEMapModder I don't think I need an else

**Comment by Addison118** [comment]
@PEMapModder *

File: src/legionpe/theta/command/session/ChannelCommand.php (hunk 2 above)
Comment (PEMapModder) [inline src/legionpe/theta/command/session/ChannelCommand.php]: You won't join channel if you are team or local.

**Comment by PEMapModder** [comment]
Please be aware that there is an **edit** button.

**Comment by Addison118** [comment]
I know well that there is an edit button. Now why do I need an else?

**Comment by PEMapModder** [comment]
Because you didn't return.
If you don't return, `/ch team` and `/ch local` will still run this code. i.e. they will still `joinChannel("local/team")`.

**Pull request closed without merging**